onnxruntime/server/util.h

```diff
@@ -41,6 +41,11 @@ class MemBufferArray {
 // Generate protobuf status from ONNX Runtime status
 google::protobuf::util::Status GenerateProtobufStatus(const onnxruntime::common::Status& onnx_status, const std::string& message);
 
+namespace internal {
+std::string InternalRequestId();
+extern const char* REQUEST_HEADER;
+extern const char* CLIENT_REQUEST_HEADER;
```

Comment:
Why does this need to be extern for linking? Also, use string please.

Reply:
it's not in a class so it has to be extern, otherwise it'll be redefined.